charts/emby/1.6.3/Chart.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: emby
+version: 1.6.3
+appVersion:  4.5.4.0
+description: Emby Server is a home media server
```

Comment:
try keeping the upstreamVersion here, but comment it out (see other examples without upstream)

Reply:
Done